src/freeze-dry/index.test.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-env jest */
+/* eslint import/namespace: "off" */
+
+import freezeDry from 'src/freeze-dry/index'
+import * as inlineStyles from 'src/freeze-dry/inline-styles'
+import * as removeScripts from 'src/freeze-dry/remove-scripts'
+import * as inlineImages from 'src/freeze-dry/inline-images'
+import * as fixLinks from 'src/freeze-dry/fix-links'
+
+beforeAll(() => {
+    inlineStyles.default = jest.fn()
+    removeScripts.default = jest.fn()
+    inlineImages.default = jest.fn()
+    fixLinks.default = jest.fn()
+})
+
+describe('freezeDry', () => {
+    test('should call the whenAllSettled with all the jobs', async () => {
+        expect.assertions(4)
+        await freezeDry()
+        expect(inlineStyles.default).toHaveBeenCalled()
+        expect(removeScripts.default).toHaveBeenCalled()
+        expect(inlineImages.default).toHaveBeenCalled()
+        expect(fixLinks.default).toHaveBeenCalled()
+    })
```

Comment:
@Treora These tests are called unit tests. The nature of the tests are such that they are not supposed to use an external resources and just check if the code traces the steps as it is supposed. The tests are designed in such a manner because of the no external dependency policy, which means if a change in the code takes place it should only effect the test that runs on it.
I think what you are referring to are integrations tests, the testing team will be working on them next and they will be soon implemented.